addons/mail/static/tests/mock_server/mock_models/res_role.js

```diff
@@ -0,0 +1,15 @@
+import { fields, models } from "@web/../tests/web_test_helpers";
+
+export class ResRole extends models.ServerModel {
+    _name = "res.role";
+
+    name = fields.Char();
+
+    /** @param {object} ResRole */
+    _to_store(ids, store) {
+        for (const roles of this.browse(ids)) {
+            const [data] = this._read_format(roles.id, ["name"], false);
+            store.add(this.browse(roles.id), data);
+        }
+    }
```

Comment:
No to store in python so it shouldn't be here, just move this code into the other function.